src/App.tsx

```diff
@@ -1,60 +1,187 @@
-import classNames from 'classnames';
+/* eslint-disable @typescript-eslint/indent */
 
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { useEffect, useState } from 'react';
 
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { PostDetails } from './components/PostDetails';
+import { PostsList } from './components/PostsList';
 import { Loader } from './components/Loader';
+import { Comment, CommentData } from './types/Comment';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import {
+  addNewComment,
+  deleteComment,
+  getUserPostComments,
+  getUserPosts,
+  getUsers,
+} from './api/post';
 
-export const App = () => (
-  <main className="section">
-    <div className="container">
-      <div className="tile is-ancestor">
-        <div className="tile is-parent">
-          <div className="tile is-child box is-success">
-            <div className="block">
-              <UserSelector />
-            </div>
+export const App = () => {
```

Comment:
You can group your imports to make them more readable. For example, you can first import libraries (like `react`, `classnames`, etc.), then styles, then components, and lastly utilities and types.